### ISSUE TITLE
about noise_map

### ISSUE THREAD

**Issue by sunyclj** [ChenyangQiQi/BSVD#10]
In bsvd_arch.py , 
if noise_map != None:
        input = torch.cat([input, noise_map], dim=2)
please, what does the noise_map do?
